topics/transcriptomics/tutorials/rna-seq-bash-star-align/tutorial.md

```diff
@@ -0,0 +1,367 @@
+{: .comment}
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+> <comment-title></comment-title>
+>
+> This tutorial is significantly based on Galaxy's ["Reference-based RNA-Seq data analysis"](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/ref-based/tutorial.html) tutorial.
+>
+{: .comment}
+
+# Data Upload
+
+At first, we need to put our hands on our data. In this tutorial we will use the files for 2 out of the 7 samples to demonstrate how to calculate read counts (a measure of the gene expression) from FASTQ files (quality control, mapping, read counting). The FASTQ files for the other 5 samples, are provided, in case you want to reproduce the whole analysis later.
+The "Data Upload" process is the only one in this tutorial that takes place directly into Galaxy's interface.
+
+
+> <hands-on-title>Data upload</hands-on-title>
+>
+>    Import the FASTQ file pairs from [Zenodo]({{ page.zenodo_link }}) or a data library:
+>    - `GSM461177` (untreated): `GSM461177_1` and `GSM461177_2`
+>    - `GSM461180` (treated): `GSM461180_1` and `GSM461180_2`
+>
+>    ```text
+>    {{ page.zenodo_link }}/files/GSM461177_1.fastqsanger
+>    {{ page.zenodo_link }}/files/GSM461177_2.fastqsanger
+>    {{ page.zenodo_link }}/files/GSM461180_1.fastqsanger
+>    {{ page.zenodo_link }}/files/GSM461180_2.fastqsanger
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+>    > <comment-title></comment-title>
+>    >
+>    > Note that these are the full files for the samples and ~1.5Gb each so it may take some minutes to import.
+>    >
+>    > For a quicker run-through of the FASTQ steps a small subset of each FASTQ file (~5Mb) can be found here on [Zenodo]({{ page.zenodo_link }}):
+>    >
+>    > ```text
+>    > {{ page.zenodo_link }}/files/GSM461177_1_subsampled.fastqsanger
+>    > {{ page.zenodo_link }}/files/GSM461177_2_subsampled.fastqsanger
+>    > {{ page.zenodo_link }}/files/GSM461180_1_subsampled.fastqsanger
+>    > {{ page.zenodo_link }}/files/GSM461180_2_subsampled.fastqsanger
+>    > ```
+>    >
+>    {: .comment}
+>
+>    >Change the datatype from `fastqsanger` to `fastq`.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="fastq.gz" %}
+>
+>
+>    We also need to import two more files, essential for the alignment operation (and basically every alignment procedure): the organism's reference genome (here *D. melanogaster*) and the organism's gene annotation.
+>    Those can be aquired directly through data library as described above. For this tutorial we are going to use the files [dm3.fa](https://usegalaxy.eu/libraries/folders/Fdf740ff7e86f2efb/dataset/8ab324b9329be9fc) and [Drosophila_melanogaster.BDGP6.87.gtf (dm6)](https://usegalaxy.eu/libraries/folders/F30cab321d898d2fb/dataset/02c5f7fcdb6bf41f).

```

Comment:
It does not make sense to use a fasta for dm3 and use a gtf for dm6.
You can get dm6 fasta at https://hgdownload.soe.ucsc.edu/goldenPath/dm6/bigZips/dm6.fa.gz

Reply:
You are absolutely right! However, I thing it would be better to "downgrade" the gtf file (to dm3) in order to have every piece of data available in Galaxy's library.

Reply:
I think you can ask to have dm6.fa in a data library. For me it makes more sense.

Reply:
On second thought, it is instructed to fetch data from Zenodo a few lines above in the same tutorial, so my previous argument is invalid. Shall I apply the change you proposed in your previous comment, as quoted? 

> It does not make sense to use a fasta for dm3 and use a gtf for dm6.
> You can get dm6 fasta at https://hgdownload.soe.ucsc.edu/goldenPath/dm6/bigZips/dm6.fa.gz